tests/v1/engine/test_async_llm.py

```diff
@@ -175,6 +175,7 @@ async def test_abort(monkeypatch: pytest.MonkeyPatch,
                     f"expected {expected_tokens}")
 
         # Make sure all aborted requests were really aborted.
+        await asyncio.sleep(0.1)
```

Comment:
@KubeKyrie could you explain why this is needed? We already yield after each of the `cancel()` calls above